ada/testsuite/tests/name_resolution/unint_attrs/test_unint_attrs.adb

```diff
@@ -23,8 +23,11 @@ begin
       H  : constant Integer := Volt'Aft;
       I  : constant Integer := Volt'Fore;
       J  : constant Natural := M'Modulus;
+      L  : constant Natural := Standard'Word_Size;
```

Comment:
Please make this a valid file by not naming every variable `L` :stuck_out_tongue: 

Also, adding the new variables at the end will make for easier to reviews test baseline diffs.

Reply:
Thanks, Done!